app/bundles/LeadBundle/Entity/LeadFieldRepository.php

```diff
@@ -192,6 +192,23 @@ public function compareValue($lead, $field, $value, $operatorExpr)
                 )
                   ->setParameter('lead', (int) $lead)
                   ->setParameter('value', $value);
+            } elseif ($operatorExpr === 'in' || $operatorExpr === 'notIn') {
+                $expr = $q->expr()->andX(
+                    $q->expr()->eq('l.id', ':lead')
+                );
+
+                $value = trim($value, "'");
+                if (substr($operatorExpr, 0, 3) === 'not') {
+                    $operator = 'NOT REGEXP';
+                } else {
+                    $operator = 'REGEXP';
+                }
+                $expr->add(
+                    'l.'.$field." $operator '\\\\|?$value\\\\|?'"
```

Comment:
This will break if $value has any of MySQL's regex special characters.

Reply:
Added the same escape cleaner like https://github.com/mautic/mautic/blob/ad1e4481d0db3374e14172c64736196c8e7b76ef/app/bundles/LeadBundle/Entity/LeadListRepository.php#L1681